docs/modules/install/con-install-product-downloads.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// assemblies/assembly-operator-install.adoc
+
+[id='downloads-{context}']
+= Downloading Kroxylicious release artifacts
+
+[role="_abstract"]
+To use install files to install Kroxylicious, download and extract the files from the {OperatorDownload}.
```

Comment:
I've suggested to @tombentley on #2305 that we include a classifier into the tgz name so that the user can identify it easily amongst the release artefacts.